packages/connect/src/connector/index.ts

```diff
@@ -13,10 +13,10 @@ const isExtensionPresent =
   !!document.getElementById(DOM_ELEMENT_ID)
 
 /**
- * Returns a {SubstrateConnector} that connects to chains, either through the substrate-connect
+ * Returns a {ScClient} that connects to chains, either through the substrate-connect
  * extension or by executing a light client directly from JavaScript, depending on whether the
  * extension is installed and available.
  */
-export const getConnectorClient = isExtensionPresent
-  ? eGetConnectorClient
-  : slGetConnectorClient
+export const createScClient = isExtensionPresent
+  ? ecreateScClient
+  : slcreateScClient
```

Comment:
As per previous comment:
```suggestion
  ? extensionScClient
  : smoldotScClient
```